docs/JavaScriptEnvironment.md

```diff
@@ -45,3 +47,4 @@ ES7
 
 * [Object Spread](https://github.com/sebmarkbage/ecmascript-rest-spread): `var extended = { ...obj, a: 10 };`
 * [Function Trailing Comma](https://github.com/jeffmo/es-trailing-function-commas): `function f(a, b, c,) { }`
+* [Async Functions](https://github.com/tc39/ecmascript-asyncawait): `async function doStuffAsync(){ }; const foo = await doStuffAsync()`;
```

Comment:
Can you move the await call inside of the async function? Sometimes people think they can use await outside of an async fn.